README.md

```diff
@@ -383,6 +383,14 @@ Add following keybindings to config:
 ```
 Same approach can be used for other `vim-sexp` keybindings (e.g. `<localleader>e[`) with cursor placement or without.
 
+### `vim-sexp` splice sexp
+
+```lua
+  ["<localleader>@"] = {
+    paredit.unwrap.unwrap_form_under_cursor,
+    "Splice sexp",
+  },
+```
```

Comment:
Why not add this as a default keybinding?

Reply:
Sure, will add.